city_metrix/layers/albedo.py

```diff
@@ -89,20 +107,25 @@ def calc_s2_albedo(image):
                 'SWIR1': image.select('B11'),
                 'SWIR2': image.select('B12')
             }
-            return image.expression(S2_ALBEDO_EQN, config).double().rename('albedo')
+
+            albedo = image.expression(S2_ALBEDO_EQN, config).double().rename('albedo')
+
+            return albedo
 
         ## S2 MOSAIC AND ALBEDO
         dataset = get_masked_s2_collection(ee.Geometry.BBox(*bbox), self.start_date, self.end_date)
         s2_albedo = dataset.map(calc_s2_albedo)
         albedo_mean = s2_albedo.reduce(ee.Reducer.mean())
 
-        data = (get_image_collection(
-            ee.ImageCollection(albedo_mean), bbox, self.spatial_resolution, "albedo")
-                .albedo_mean)
+        albedo_mean_ic = ee.ImageCollection(albedo_mean)
+        data = get_image_collection(
+            albedo_mean_ic,
+            bbox,
+            self.spatial_resolution,
+            "albedo"
+        ).albedo_mean
```

Comment:
Do we want the ) on the line above in cases like this?

Reply:
I think it's comprehensible as is.